milatools/cli/commands.py

```diff
@@ -684,17 +1013,22 @@ def _standard_server(
         proc.kill()
 
 
-@tooled
-def _find_allocation(remote, job_name="mila-tools"):
-    # Node to connect to
-    node: Option = default(None)
+def _find_allocation(
+    remote,
+    node: str | None,
+    job: str | None,
+    alloc: Sequence[str],
+    job_name: str = "mila-tools",
+):
+    # # Node to connect to
+    # node: Option = default(None)
 
-    # Job ID to connect to
-    job: Option = default(None)
+    # # Job ID to connect to
+    # job: Option = default(None)
 
-    # Extra options to pass to slurm
-    # [nargs: --]
-    alloc: Option = default([])
+    # # Extra options to pass to slurm
+    # # [nargs: --]
+    # alloc: Option = default([])
```

Comment:
Remove commented lines.

Reply:
Fixed in 39a4d59d38f703d2a2bd4c4b762fae70a45ef464